data/context/standard_test.go

```diff
@@ -6,19 +6,28 @@ import (
 
 	"github.com/tidepool-org/platform/data"
 	"github.com/tidepool-org/platform/data/context"
+	"github.com/tidepool-org/platform/log/test"
 	"github.com/tidepool-org/platform/service"
 )
 
 var _ = Describe("Standard", func() {
-	Context("New", func() {
+	It("NewStandard returns an error if logger is nil", func() {
+		standard, err := context.NewStandard(nil)
+		Expect(standard).To(BeNil())
+		Expect(err).To(HaveOccurred())
```

Comment:
Ah, good point. However, I just looked and there are a whole bunch of these `HaveOccurred()`. I'll convert all of them at once in another PR.